openrouteservice-api-tests/src/test/java/org/heigit/ors/services/routing/ResultTest.java

```diff
@@ -764,7 +764,7 @@ public void testExtrasDetails() {
 				.when().log().ifValidationFails()
 				.get(getEndPointName());
 
-		response.then()
+		response.then().log().all()
```

Comment:
Probably better here to only log if the validation fails to save cluttering up the output of tests

Reply:
forgot to remove the line...